src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -193,6 +184,7 @@ class WorkspaceService(userInfo: UserInfo, dataSource: DataSource, containerDAO:
         }
 
         gcsDAO.deleteBucket(userInfo, workspaceContext.workspace.workspaceId).map { _ =>
+          containerDAO.authDAO.deleteWorkspaceAccessGroups(workspaceContext.workspace, txn)
```

Comment:
So it does. I'm picking nits now, but I'd recommend replacing references to `groupAccessLevelsAscending` in that code with `WorkspaceAccessLevels.all` so they don't go out of sync if we add more access levels. If you're wary of requesting deletion of buckets that don't exist, at least move the definition of `groupAccessLevelsAscending` next to `all` so they're in the same place.